controllers/suite_test.go

```diff
@@ -74,17 +81,30 @@ func TestAPIs(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	done := make(chan interface{})
-	go func() {
 
+	// log msgs from reconciler to a log file
+	var f io.Writer
+	f, err := os.Create("/tmp/ocs-osd-deployer.log")
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Please take a look at the ginkgo `-v` option, that according to docs should stream all of the writer output to stdout

Reply:
- Unfortunately I don't see any log o/p in stdout with this change 
``` diff
# gd Makefile 
diff --git a/Makefile b/Makefile
index 0b6646a..c61f7b3 100644
--- a/Makefile
+++ b/Makefile
@@ -45,7 +45,7 @@ ENVTEST_ASSETS_DIR = $(shell pwd)/testbin
 test: generate fmt vet manifests
        mkdir -p $(ENVTEST_ASSETS_DIR)
        test -f $(ENVTEST_ASSETS_DIR)/setup-envtest.sh || curl -sSLo $(ENVTEST_ASSETS_DIR)/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/v0.6.3/hack/setup-envtest.sh
-       source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out
+       source $(ENVTEST_ASSETS_DIR)/setup-envtest.sh; fetch_envtest_tools $(ENVTEST_ASSETS_DIR); setup_envtest_env $(ENVTEST_ASSETS_DIR); go test ./... -coverprofile cover.out -ginkgo.v
 
 # Build manager binary
 manager: generate fmt vet```

Reply:
Ok, so we will go with your original method